src/main/java/com/google/cloud/tools/app/deploy/process/LoggingOutputHandler.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.tools.app.deploy.process;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LoggingOutputHandler implements OutputHandler {
+
+  private final Logger log;
+  private final Level level;
+
+  public LoggingOutputHandler(String logName, Level logLevel) {
```

Comment:
Its not in the API specifically, but it's just a default implementation. Unless I'm using as a default somewhere?